src/config.rs

```diff
@@ -606,6 +612,7 @@ pub struct ConfigFile {
     pub device: Option<ConfigFileDevice>,
 
     pub ip: Option<String>,
+    pub advertise_addresses: Option<Vec<String>>,
```

Comment:
Please also add to Args to allow --advertise-addresses

Reply:
Done (latest commit)